src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassFanOutComplexityCheck.java

```diff
@@ -51,6 +51,9 @@ public ClassFanOutComplexityCheck() {
             TokenTypes.PACKAGE_DEF,
             TokenTypes.IMPORT,
             TokenTypes.CLASS_DEF,
+            TokenTypes.EXTENDS_CLAUSE,
+            TokenTypes.IMPLEMENTS_CLAUSE,
+            TokenTypes.ANNOTATION,
```

Comment:
@romani 
Please confirm that we want these new tokens to be in `getRequiredTokens`, we might only need them in `getAcceptableTokens` and `getDefaultTokens`